great_expectations/render/renderer/column_section_renderer.py

```diff
@@ -55,34 +55,34 @@ def render(cls, evrs, section_name=None, column_type=None):
             column = section_name
 
         content_blocks = []
-        cls._render_header(evrs, content_blocks, column_type)
-        # cls._render_column_type(evrs, content_blocks)
-        cls._render_overview_table(evrs, content_blocks)
-        cls._render_quantile_table(evrs, content_blocks)
-        cls._render_stats_table(evrs, content_blocks)
-        cls._render_histogram(evrs, content_blocks)
-        cls._render_values_set(evrs, content_blocks)
-        cls._render_bar_chart_table(evrs, content_blocks)
 
-        # cls._render_statistics(evrs, content_blocks)
-        # cls._render_common_values(evrs, content_blocks)
-        # cls._render_extreme_values(evrs, content_blocks)
-
-        # cls._render_frequency(evrs, content_blocks)
-        # cls._render_composition(evrs, content_blocks)
-
-        # cls._render_expectation_types(evrs, content_blocks)
-        # cls._render_unrecognized(evrs, content_blocks)
-
-        cls._render_failed(evrs, content_blocks)
+        content_blocks.append(cls._render_header(evrs, column_type))
```

Comment:
(discussed offline) : some content_block rendering methods in other classes return 2 or more blocks. Maybe return a list of blocks instead of just a block?